sql/core/src/test/resources/sql-tests/inputs/transform.sql

```diff
@@ -0,0 +1,49 @@
+-- Test data.
+CREATE OR REPLACE TEMPORARY VIEW t1 AS SELECT * FROM VALUES
+('a'), ('b'), ('v')
+as t1(a);
+
+CREATE OR REPLACE TEMPORARY VIEW t2 AS SELECT * FROM VALUES
```

Comment:
we need the two tests for the following tests?

Reply:
> we need the two tests for the following tests?

Can merge this, Done